state-indexer/src/configs.rs

```diff
@@ -94,8 +94,8 @@ impl Opts {
 
     pub fn rpc_url(&self) -> &str {
         match &self.chain_id {
-            ChainId::Mainnet(_) => "https://rpc.mainnet.near.org",
-            ChainId::Testnet(_) => "https://rpc.testnet.near.org",
+            ChainId::Mainnet(_) => "https://archival-rpc.mainnet.near.org",
```

Comment:
I believe it's a good time to stop using the hardcoded values in favor of the configuration provided as we did for RPC-server. Please add regular and archival RPC URLs to the config and use the required one here.

Reply:
I agree with you. This will be improved in the next pull request for configuration improvement. Work is already underway here https://github.com/near/read-rpc/pull/133

Reply:
Why do we need to use Archival RPC here then?

Reply:
Yes. I did some testing and the epoch history can only be retrieved from the archive node.